opal/group_remote_info.go

```diff
@@ -7,10 +7,11 @@ import (
 )
 
 // NOTE: Unfortunately, terraform go-sdk does not support nested object types natively. The only work-around
-//       is to have a schema.ListType with MaxItems=1 as a layer of indirection in between each level of nesting.
-//       This makes the implementation ugly, but it's thankfully mostly hidden from the client. If nested objects
-//       are ever natively supported it in the SDK, we should be able to update our code without a need for
-//       change in the HCL of clients.
+//
+//	is to have a schema.ListType with MaxItems=1 as a layer of indirection in between each level of nesting.
```

Comment:
this diff looks unintentional?

Reply:
For some reason go fmt did not like how I commented this in the first place and keeps changing it to that. I changed it to something more sane.